wremnants/datasets/datagroups.py

```diff
@@ -482,7 +483,7 @@ def sum_and_unc(h):
                 columns=["Process", "Yield", "Uncertainty"])
         return df
 
-    def readHist(self, baseName, proc, syst, scaleOp=None, forceNonzero=True, scaleToNewLumi=-1):
+    def readHist(self, baseName, proc, group, syst, scaleOp=None, forceNonzero=True, scaleToNewLumi=-1):
```

Comment:
The function is already defined in datagroups.py with only small differences, is it possible to use the function from the mother class here instead?

Reply:
The small differences between those functions are necessary, particularly for the fakes. This might change when we have the new functionalities for the unfolding

Reply:
It really would be good to reduce code duplication, but I guess we can do this with a later PR.

Reply:
Once David finishes his PR with the new unfolding functionalities, we can remove this duplicate function.